Source/WebKit/WebProcess/com.apple.WebProcess.sb.in

```diff
@@ -1122,7 +1122,12 @@
 
 ; Needed for [CAContext remoteContextWithOptions]
 #if HAVE(SANDBOX_STATE_FLAGS) && PLATFORM(MAC) && __MAC_OS_X_VERSION_MIN_REQUIRED >= 140000
-(with-filter (require-not (lockdown-mode))
+(with-filter
+    (require-all
+        (require-not (lockdown-mode))
+        (require-any
+            (require-not (state-flag "UnifiedPDFEnabled"))
+            (require-not (state-flag "BlockIOKitInWebContentSandbox"))))
```

Comment:
In lockdown mode, if Unified PDF is enabled and IOKit is blocked, should we grant sandbox extension so that PDF can render correctly?

Reply:
That is a good point! In Lockdown Mode, we should continue the previous behavior of always blocking CARenderServer.

Thanks for reviewing!